backend/src/main/java/pl/edu/agh/io/backend/controllers/VisualizerController.java

```diff
@@ -6,7 +6,7 @@
 import pl.edu.agh.io.backend.entities.config.JsonConfig;
 
 @RestController
-@CrossOrigin(origins = "http://localhost:4200")
+//@CrossOrigin(origins = "http://localhost:4200")
```

Comment:
To jednak powinno być odkomentowane
```suggestion
@CrossOrigin(origins = "http://localhost:4200")
```